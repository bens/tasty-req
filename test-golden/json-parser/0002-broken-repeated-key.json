{ "xxx": 1,
  "xxx": 2
}
