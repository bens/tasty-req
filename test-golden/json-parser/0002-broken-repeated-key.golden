test-golden/json-parser/0002-broken-repeated-key.json:3:1:
  |
3 | }
  | ^
Repeated object keys: ["xxx"]
